sdk/eventhub/Azure.Messaging.EventHubs/stress/src/Infrastructure/EnvironmentReader.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Azure.Core;
+
+namespace Azure.Messaging.EventHubs.Stress;
```

Comment:
At some indeterminate point in the future, we should probably look to shift the rest of the files to this form.   Very low priority, but I love that we can reduce the nesting.